draft-ietf-quic-recovery.md

```diff
@@ -1319,7 +1319,7 @@ window.
 ~~~
    CongestionEvent(sent_time):
      // Start a new congestion event if the sent time is larger
-     // than the start time of the previous recovery epoch.
+     // than the start time of the previous congestion event.
```

Comment:
Suggestion: "... if the packet was sent before the last congestion event started."